tests/src/python/test_qgsproject.py

```diff
@@ -825,6 +822,19 @@ def test_removeMapLayerById(self):
         QgsProject.instance().removeMapLayer(l3.id())
         self.assertFalse(sip.isdeleted(l3))
 
+        # ==== do it again with list
+        l1 = createLayer("test")
+        l2 = createLayer("test2")
+
+        QgsProject.instance().addMapLayers([l1, l2])
+        self.assertEqual(QgsProject.instance().count(), 2)
+
+        # remove layers
+        l1_id = l1.id()
+        l2_id = l2.id()
+        QgsProject.instance().removeMapLayers([l1_id, l2_id])
+        self.assertEqual(QgsProject.instance().count(), 0)
+
```

Comment:
Add a test with mixed layers and strings, and non layer/string types

Reply:
Done.